### PR TITLE
develop 2

### PR DESCRIPTION
**Pull request by sjuny**
please review

### CODE REVIEW

File: src2.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-dev2
\ No newline at end of file
+dev2
+dev2-2
```
Comment (sjuny) [inline src2.txt]: modify here

File: src2.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-dev2
\ No newline at end of file
+dev2
```
Comment (sjuny) [inline src2.txt]: test

File: src2.txt (hunk 1 above)
Comment (sjuny) [inline src2.txt]: test

File: src2.txt (hunk 2 above)
Comment (sjuny) [inline src2.txt]: start modify

Comment (sjuny) [reply src2.txt]: answer 1

File: src1.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 aaaaaaa
```
Comment (sjuny) [inline src1.txt]: test

File: src2.txt (hunk 2 above)
Comment (sjuny) [inline src2.txt]: test

Comment (sengtor) [inline src2.txt]: test review

Comment (sjuny) [reply src2.txt]: OK

File: src2.txt [hunk 4]
```diff
@@ -1 +1,2 @@
-dev2
```
Comment (sengtor) [inline src2.txt]: test3

File: src1.txt (hunk 3 above)
Comment (sengtor) [inline src1.txt]: test

Comment (sengtor) [inline src1.txt]: test3

Comment (sjuny) [reply src1.txt]: ok4

**Pull request merged**